maintenance_equipment_contract/models/maintenance_equipment.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class MaintenanceEquipment(models.Model):
+
+    _inherit = "maintenance.equipment"
+
+    contract_ids = fields.Many2many("contract.contract", string="Contracts")
+    contract_count = fields.Integer(compute="_compute_contract_count",)
+
+    @api.depends("contract_ids")
```

Comment:
Since `contract_count` is a non-stored computed field, is actually this decorator applied?

Reply:
If I remove it the test will fail. It can probably be solved with a `.refresh()` but I think it does not hurt to keep the depends and may give some unexpected behaviour in some cases